src/public/js/components/sideBar/component/Code/index.js

```diff
@@ -49,13 +49,9 @@ export default class extends React.Component {
       <div className={classNames('Code', { limitedHeight })} ref={el => (this.codeRef = el)}>
         <SyntaxHighlighter
           language={language}
-          style={atomOneLight}
+          style={vs}
           showLineNumbers={true}
           wrapLines={true}
-          customStyle={{
```

Comment:
is there is a way to provide custom styles to SyntaxHighlighter in this version? Need to be able to control that as it was done before

Reply:
yes it is. I removed this part only because it conflicted with the new theme I picked.